### PR TITLE
Small points for week2

### PR DESCRIPTION
**Pull request by nkuik**
Went through as much as I could (at least all the live coding, exercises, and homework template). It looks great! I feel much more confident going into week 2 😃

Just some small points. Let me know what you think.

### CODE REVIEW

File: week2/exercise-template/readme.md [hunk 1]
```diff
@@ -1,69 +1,28 @@
-<p align="center">
```
Comment (nkuik) [inline week2/exercise-template/readme.md:1]: Seems like this readme is from homework week 3?

File: week2/teacher-live-coding/src/backend/middleware.js [hunk 2]
```diff
@@ -2,7 +2,6 @@
 // Take a look inside the folder!
 const express = require("express");
 const app = express();
-const miniRouter = require("./mini-router");
```
Comment (nkuik) [inline week2/teacher-live-coding/src/backend/middleware.js:5]: `miniRouter` isn't used here. Not sure if it is supposed to be eventually used here in the live coding?

File: week2/exercise-template/readme.md (hunk 1 above)
Comment (benna100) [reply week2/exercise-template/readme.md:1]: Ahh yes, good point!

**Review by benna100 — APPROVED** [review]
Good changes, thanks :)

**Pull request merged**